build.gradle

```diff
@@ -268,7 +268,7 @@ project('test:testcommon') {
         compile group: 'commons-io', name: 'commons-io', version: commonsioVersion
         compile group: 'io.netty', name: 'netty-all', version: nettyVersion
         compile group: 'org.apache.curator', name: 'curator-test', version: apacheCuratorVersion, withoutLogger
-        compile group: 'com.google.code.gson', name: 'gson', version: gsonVersion
+        compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: jacksonVersion
```

Comment:
I suggest  you to give a try to https://github.com/FasterXML/jackson-jr

jackson-databind is full of CVEs due to the richness of features.

in ZooKeeper community we are evaluating to drop it, as many times we have to perform upgrades.

I don't know if in Pravega project there are automatic checks about CVEs but as far as I know many open source project consumers take great care of this stuff

personally I have never tried jackson-jr, but it may be worth to take a look

Reply:
@eolivelli thanks for your sharing your thoughts and suggestion. I think it's a good idea to explore if we should use `jackson-jr` instead.

That being said, we use `jackson-databind` elsewhere in the code too. And, `jackson-jr` would be a net new library, should we choose to use it. Therefore, I think it might be a good idea to explore doing it separately of this PR, after fully considering all the implications.  

As for security vulnerabilities, yes we perform a number of checks with tools like dependency analyzers, vulnerability scanners, etc. regularly, and address the issues found, regularly.

Reply:
Sure

Reply:
Thanks.